sirepo/template/genesis.py

```diff
@@ -327,6 +358,15 @@ def _get_frame_counts(run_dir):
     return res
 
 
+def _is_text_file(path):
+    try:
+        pkio.read_text(path)
```

Comment:
If the files are large, this is expensive (reads the whole file). If the file is not large, probably fine.

we may want to consider binaryornot, which is based on Perl's algorithm.

Reply:
Yes, I looked at binaryornot but we do not currently have it as part of the base install, and genesis input files are not overly large.